frontend/components/CompanyList.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div
+    class="shadow ring-1 ring-black ring-opacity-5 md:rounded-lg overflow-x-hidden overflow-y-scroll"
+  >
+    <table class="min-w-full divide-y divide-gray-300">
+      <tbody class="divide-y divide-gray-200 bg-white overflow-hidden py-5">
+        <tr v-for="person in people" :key="person.email">
```

Comment:
I think we can change it to:
```suggestion
        <tr v-for="company in companies" :key="company.email">
```

Remember to change variables in the whole file.

Reply:
Names of variables were change fit project